odoo/tools/xml_utils.py

```diff
@@ -340,3 +340,96 @@ def validate_xml_from_attachment(env, xml_content, xsd_name, reload_files_functi
 def find_xml_value(xpath, xml_element, namespaces=None):
     element = xml_element.xpath(xpath, namespaces=namespaces)
     return element[0].text if element else None
+
+
+def dict_to_xml(node, nsmap={}, template=None, render_empty_nodes=False, tag=None, path=None):
```

Comment:
When you have many parameters, forcing most of them to be kw-only improves readability and future refactoring.
```suggestion
def dict_to_xml(node, *, nsmap={}, template=None, render_empty_nodes=False, tag=None, path=None):
```